src/api/is-device-supported.js

```diff
@@ -0,0 +1,15 @@
+import getDeviceSize from '../utils/get-device-size';
+
+
+/**
+ * Checks if the device is supported as the target for the `resize` API method.
+ * @function
+ * @async
+ * @name isDeviceSupportedAsResizeTarget
+ * @param {string} deviceName - Specifies the name of the target device. You can use the values specified in the Device Name column of [this table]{@link http://viewportsizes.com/}.
+ */
```

Comment:
> Specifies the name of the target device. You can use the values specified in the Device Name column of [this table]{@link http://viewportsizes.com/}.

So you suggest to use device names from http://viewportsizes.com/. As far as I understand, this function returns `true` for all these devices. So what's the purpose?

This function is good if you can't use http://viewportsizes.com/ for reference.

Did I get it right?